### PR TITLE
Parse users

### PR DESCRIPTION
**Pull request by sturmer-eha**
added goodquestion:parse as a separate global package

### CODE REVIEW

File: app/lib/models.coffee [hunk 1]
```diff
@@ -0,0 +1 @@
+@Survey = Parse.Object.extend 'Survey'
```
Comment (jgoley) [inline app/lib/models.coffee]: This file was moved to the imports directory.

**Comment by dan-nyanko** [comment]
Not sure why this branch has package.json with `"start": "meteor -p 13000 --settings settings.json"` but the file is missing.  Either way, I've been using an uncommitted file called `settings.json.local` for testing that has API keys, passwords, etc.  Probably should just .gitignore `settings.json.local` or similar and we can all just use that without committing to version control.  What do you think?

**Comment by dan-nyanko** [comment]
I'm getting error initializing Parse.  `Error: You need to call Parse.initialize before using Parse.`

**Comment by jgoley** [comment]
There must have been a merge conflict mishap at some point. I think the npm command is supposed to read `"start": "meteor -p 13000 --settings settings-dev.json"` and `settings-dev.json` in gitignore

**Comment by jgoley** [comment]
The initialize error was due the initialized Parse object being overwritten by the lingering requires. I removed those and fixed the issue.

**Comment by jgoley** [comment]
Removed one more require in the openam package and users are creating without error now.

**Pull request merged**